kafka-ui-react-app/src/components/common/JSONViewer/JSONViewer.tsx

```diff
@@ -1,36 +1,41 @@
 import React from 'react';
-import JSONEditor from 'components/common/JSONEditor/JSONEditor';
+import Editor from 'components/common/Editor/Editor';
+import { SchemaType } from 'generated-sources';
 
 import { StyledWrapper } from './StyledWrapper.styled';
 
 export interface FullMessageProps {
   data: string;
+  schemaType?: string;
   maxLines?: number;
 }
 
-const JSONViewer: React.FC<FullMessageProps> = ({ data, maxLines }) => {
+const getSchemaValue = (data: string, schemaType?: string) => {
+  if (schemaType === SchemaType.JSON || schemaType === SchemaType.AVRO) {
+    return JSON.stringify(JSON.parse(data), null, '\t');
+  }
+  return data;
+};
+const JSONViewer: React.FC<FullMessageProps> = ({
```

Comment:
```js
const JSONViewer: React.FC<FullMessageProps> = ({
```
Is JSONViewer a correct name?

Reply:
@Motherships as we changed JSONEditor to Editor I guess we can have EditorViewer
Do you have a better option?

Reply:
It's okay, think
`EditorViewer` gives understanding that i coupled with `Editor`